apps/protocol-frontend/src/components/Sidebar.tsx

```diff
@@ -206,6 +208,8 @@ const Sidebar = () => {
               icon={FiMessageSquare}
               linkTo={FEEDBACK_LINK}
             />
+            <NavButton label="Docs" icon={FiBook} linkTo={DOCS_LINK} />
+            // todo add the navbutton for the FIbook icon
```

Comment:
Let's remove this comment since you added it!

Reply:
Nice, totally